src/network/connection.js

```diff
@@ -69,6 +70,7 @@ module.exports = class Connection {
     maxInFlightRequests = null,
     instrumentationEmitter = null,
   }) {
+    this.id = id;
```

Comment:
Have to remove the semicolon here or prettier errors on `yarn test`

Reply:
Ups..good catch. Thanks!